bin/run.py

```diff
@@ -319,28 +315,18 @@ def process_run(run, p):
 
             localbar.done(got_expected, f'{result.duration:6.3f}s {result.print_verdict()}', data)
 
-            # Lazy judging: stop on the first error when:
-            # - not in verbose mode
-            if config.args.verbose or config.args.table:
-                return
-            # - the result has max priority
-            if result.verdict not in config.MAX_PRIORITY_VERDICT:
-                return
             # - for TLE, the run was aborted because the global timeout expired
-            if result.verdict == 'TIME_LIMIT_EXCEEDED' and not result.timeout_expired:
+            # TODO: What does this mean?
+            if result.verdict == Verdict.TIME_LIMIT_EXCEEDED and not result.timeout_expired:
                 return
 
-            bar.count = None
-            p.abort()
-
```

Comment:
RE `TODO: What does this mean?`:

You moved the `bar.count = None` and `p.abort()` up. If you read it like this:
```py
            if result.verdict != 'TIME_LIMIT_EXCEEDED' or result.timeout_expired:
                bar.count = None
                p.abort()
```
So, in the current implementation on `master`, we abort TLE submissions _only_ when the hard timeout expired. This new implementation also aborts submissions that run just a single millisecond over the problem's time limit.

It looks like you moved the code that aborts the remaining test case runs to https://github.com/RagnarGrootKoerkamp/BAPCtools/pull/358/files#diff-dcd3fdd39ac1d0938a32c9c9c9ef70fc8b00f9d336d0676e30f159ee0e8d2314R263-R265. Can you re-add here that TLE submissions only abort once they reach the hard timeout? :slightly_smiling_face: